src/main/webapp/components/BuilderGrid.js

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import {getField} from "./utils/getField";
+
+/**
+ * Component used as an indicator for build status.
+ * Statuses are differentiated by alternative
+ * styling rules.
```

Comment:
nit: please have one blank line between annotations and function description

Reply:
Resolved b5dd82a